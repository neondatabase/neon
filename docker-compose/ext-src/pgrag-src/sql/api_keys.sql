-- API key function tests
SELECT rag.anthropic_set_api_key('test_key');

SELECT rag.anthropic_get_api_key();

SELECT rag.openai_set_api_key('test_key');

SELECT rag.openai_get_api_key();

SELECT rag.fireworks_set_api_key('test_key');

SELECT rag.fireworks_get_api_key();

SELECT rag.voyageai_set_api_key('test_key');

SELECT rag.voyageai_get_api_key();
