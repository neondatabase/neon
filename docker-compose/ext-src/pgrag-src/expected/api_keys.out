-- API key function tests
SELECT rag.anthropic_set_api_key('test_key');
 anthropic_set_api_key 
-----------------------
 
(1 row)

SELECT rag.anthropic_get_api_key();
 anthropic_get_api_key 
-----------------------
 test_key
(1 row)

SELECT rag.openai_set_api_key('test_key');
 openai_set_api_key 
--------------------
 
(1 row)

SELECT rag.openai_get_api_key();
 openai_get_api_key 
--------------------
 test_key
(1 row)

SELECT rag.fireworks_set_api_key('test_key');
 fireworks_set_api_key 
-----------------------
 
(1 row)

SELECT rag.fireworks_get_api_key();
 fireworks_get_api_key 
-----------------------
 test_key
(1 row)

SELECT rag.voyageai_set_api_key('test_key');
 voyageai_set_api_key 
----------------------
 
(1 row)

SELECT rag.voyageai_get_api_key();
 voyageai_get_api_key 
----------------------
 test_key
(1 row)

